node/src/effect.rs

```diff
@@ -1662,6 +1663,20 @@ impl<REv> EffectBuilder<REv> {
             .await
     }
 
+    /// Announce that the sync process has finished.
+    pub(crate) async fn announce_finished_syncing(self)
+    where
+        REv: From<ChainSynchronizerAnnouncement>,
+    {
+        info!("announcing chain sync finished");
```

Comment:
I'm not sure we need this log line.  We already have `info!("finished chain sync to genesis");` inside the chain sync code, and the other callsite is when the user has disabled sync-to-genesis, so seeing this log message might be a bit confusing.

Reply:
Regardless, let's not put log messages into the effect builder if we can avoid it.

Reply:
Log message removed in 5b91f0f74616a4c60822d297c58f08523917ec38